remsfal-services/remsfal-ticketing/docker-compose.yml

```
@@ -4,14 +4,14 @@ services:
     container_name: minio
     environment:
       - MINIO_ROOT_USER=remsfal
-      - MINIO_ROOT_PASSWORD=remsfal
+      - MINIO_ROOT_PASSWORD=remsfaladmin
```

Excuse me. I thought we were talking about the username. For the password, `remsfalSecret` would probably be better.

```suggestion
      - MINIO_ROOT_PASSWORD=remsfalSecret
```